main.js

```diff
@@ -177,6 +177,10 @@ function createWindow () {
     }
   ];
 
+  if (process.platform === "linux") {
+    template[0].submenu[2] = { label: "Screenshot Protection Not Available On Linux", enabled: false};
+  }
```

Comment:
I don't think this is the best way to go about it -- specifically i'm talking about referencing template element 0 & submenu element 2. If a submenu gets added before the screenshot label then this will cause unintended effects.

Would be preferred to call a method from line 153 that gets the appropriate screenshot label